alfresco-transform-tika/alfresco-transform-tika/LICENSES.md

```diff
@@ -1,5 +1,4 @@
 ### Licenses
 
-* Tika is from Apache. See the license at http://www.apache.org/licenses/LICENSE-2.0 or the 
-[Apache 2.0.txt](https://github.com/Alfresco/acs-community-packaging/blob/master/distribution/src/main/resources/licenses/3rd-party/Apache%202.0.txt) 
-file placed in the root directory of the docker image.
+* Tika is from Apache. See the licence at http://www.apache.org/licenses/LICENSE-2.0 or the [Apache 2.0.txt](https://github.com/Alfresco/alfresco-community-repo/blob/master/packaging/distribution/src/main/resources/licenses/3rd-party/Apache%202.0.txt) file placed in the root directory of the docker image.
```

Comment:
Replace https://github.com/Alfresco/alfresco-community-repo ... with link to https://github.com/Alfresco/alfresco-transform-core ?

Reply:
The licence file is no in transform-core. It's still in the repo project.

Reply:
I saw it in the file changed list -> https://github.com/Alfresco/alfresco-transform-core/pull/384/files#diff-8944d537efc1e569d29a2ed0f0586340b8270ec3d9b5423c6e32108dbde1524d

Reply:
The licence files are in the top level of each *-boot project so that they are copied into the docker image.

Reply:
Thanks.

Sorry, to be picky but we should rename the new folders from /licence/ to /license/ (and fix references) to be consistent with existing US usage, eg.

https://github.com/Alfresco/alfresco-transform-core/tree/master/alfresco-transform-tika/alfresco-transform-tika-boot/src/license

Reply:
Thanks for fixing (/licenses/ is even better) :-)